kythe/go/extractors/config/wrapper/gradle_build_modifier.go

```diff
@@ -20,11 +20,11 @@ allprojects {
 `
 
 // This matches a line which sets the javac to use Kythe's javac-wrapper.sh
-var kytheMatcher = regexp.MustCompile(`\s+options\.forkOptions\.executable\ =\ '/opt/kythe/extractors/javac-wrapper.sh'`)
+var kytheMatcher = regexp.MustCompile(`\n\s+options\.forkOptions\.executable\ =\ '/opt/kythe/extractors/javac-wrapper.sh'`)
```

Comment:
I think you can get the same effect via `(?m)^\s*options\.forkOptions\.Executable\ =…`